app/services/report_service.rb

```diff
@@ -19,8 +19,9 @@ def call(source_account, target_account, options = {})
     notify_staff!
 
     if forward?
-      forward_to_origin!
-      forward_to_replied_to!
+      ReportForwardingService.new.call(@report, @source_account, {
+        forward_to_domains: @options[:forward_to_domains] || [@report.target_account.domain],
```

Comment:
If I'm understanding what this code did correctly, then if the `forward_to_domains` option wasn't specified, then it'd always forward to the target_account's domain. We may want to reassess that, and instead rely on manual forwarding — I don't know if a missing API parameter should be seen as "yes, let's forward this to a remote instance"